custom/icds_reports/utils/aggregation_helpers/distributed/agg_child_health.py

```diff
@@ -116,25 +116,27 @@ def staging_queries(self):
                 "SUM(CASE WHEN (chm.born_in_month = 1 AND (chm.nutrition_status_weighed = 1 "
                 "OR chm.low_birth_weight_born_in_month = 1)) THEN 1 ELSE 0 END)"),
             ('zscore_grading_hfa_normal',
-                "SUM(CASE WHEN chm.zscore_grading_hfa_recorded_in_month = 1 AND "
+                "SUM(CASE WHEN chm.wer_eligible=1 AND chm.zscore_grading_hfa_recorded_in_month = 1 AND "
```

Comment:
@calellowitz  I have used `wer_eligible` since we have made height_eligible same as wer_eligible and height_eligible is not stored in CHM table as a column.
Feel free to object this if you thing using wer_eligible for HFA and WFH indicators will make things confusing. I will check the height_eligiblity conditions manually.